src/main/java/seedu/eylah/expensesplitter/model/PersonAmountBook.java

```diff
@@ -110,8 +109,20 @@ public void removePerson(Person key) {
 
     @Override
     public String toString() {
-        return persons.asUnmodifiableObservableList().size() + " persons";
-        // TODO: refine later
+        final StringBuilder builder = new StringBuilder();
+        ObservableList<Person> personsList = persons.asUnmodifiableObservableList();
+        builder.append("Person : Amount\n");
+        for (Person person : personsList) {
+            int index = personsList.indexOf(person) + 1;
```

Comment:
Nice! If i understand your code correctly, this basically adds the Json parameter (amount) into personamount.json right?

Reply:
For that part, I add the field in the `JsonAdaptedPerson` class.